packages/starlight/utils/user-config.ts

```diff
@@ -126,26 +126,21 @@ const UserConfigSchema = z.object({
 
   /** Optional details about the social media accounts for this site. */
```

Comment:
Suggestion to move the examples we lost below into this comment:

```suggestion
  /**
   * Optional details about the social media accounts for this site.
   *
   * @example
   * social: {
   *   codeberg: 'https://codeberg.org/knut/examples',
   *   discord: 'https://astro.build/chat',
   *   github: 'https://github.com/withastro/starlight',
   *   linkedin: 'https://www.linkedin.com/company/astroinc',
   *   mastodon: 'https://m.webtoo.ls/@astro',
   *   threads: 'https://www.threads.net/@nmoodev',
   *   twitch: 'https://www.twitch.tv/bholmesdev',
   *   twitter: 'https://twitter.com/astrodotbuild',
   *   youtube: 'https://youtube.com/@astrodotbuild',
   * }
   */
```